functions/has_ip_address.pp

```diff
@@ -0,0 +1,10 @@
+# @summary Returns true if the client has the requested IPv4 address on some interface.
+#
+# @param ip_address
```

Comment:
Should we also support ipv6?

Reply:
Ah right, that would require updating `stdlib::has_interface_with()` to work with IpAddr objects